driver-embedded/src/main/com/mongodb/embedded/client/MongoDBCAPIHelper.java

```diff
@@ -165,4 +172,49 @@ private static void checkInitialized() {
 
     private MongoDBCAPIHelper() {
     }
+
+    /**
+     * Represents libmongodbcapi_init_params
+     */
+    public static class MongoDBCAPIInitParams extends Structure implements MongoDBCAPI.InitParams {
```

Comment:
The core representation of `libmongodbcapi_init_params`:

```
typedef struct {
    const char* yaml_config;
    uint64_t log_flags;
    libmongodbcapi_log_callback log_callback;
    void* log_user_data;
} libmongodbcapi_init_params;
```

Declaring it in the helper allows the inclusion of the `logCallback` while keeping this code protected.

JNA requires the fields of a `Structure` to be public hence the checkstyle and findbugs excludes.